src/odemis/driver/technolution.py

```diff
@@ -1402,6 +1420,21 @@ def _setCellTranslation(self, cellTranslation):
                                      "Cell translation parameter values remain unchanged." %
                                      (row, column))
 
+        # check: max cell translation <= cell complete size - effective cell size = max possible cell translation
+        max_cell_translation = max(max(cellTranslation))
```

Comment:
If `cellTranslation` was a numpy array, you could directly write `max(cellTranslation)`, otherwise, you have to explicitly iterate over each sub-tuple before taking the whole max.